### ISSUE TITLE
Cannot multiply 1 by 0 in strict mode

### ISSUE THREAD

**Issue by esigoillot** [MikeMcl/big.js#148]
Hello!

When using `Big.strict = true`, you cannot multiply `new Big('1')` with `new Big('0')`.
It generates a `TypeError: Invalid number`.

Test case:
```js
const Big = require('big.js');
Big.strict = true;
new Big('0').times(new Big('1')); // Type error
```

**Comment by MikeMcl** [comment]
Awful bug. Embarassing.

The bug happens because strict mode prevents a Big from being created from a primitive number, yet the library does just that itself internally when multiplying Big zero or multiplying a Big by zero. It also happens with division.

It's an easy fix which will be published tomorrow.

Thanks very much for the report.

**Comment by MikeMcl** [comment]
Fixed in *v6.0.3*.

**Comment by alfaproject** [comment]
@MikeMcl missed opportunity for a simple unit test?
